Automattic/vip-go-mu-plugins
# Add/admin notice dismiss

## Description

We have recently added admin-notice plugin. This change adds the ability to dismiss them.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR
1. Add a notice that shows up with dismiss identifier
1. Click dissmiss buton
1. See that notice disapeared even after reload
